packages/spotlight/Spottable/Spottable.js

```diff
@@ -182,11 +182,11 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 		}
 
 		componentWillReceiveProps (nextProps) {
-			const spottableDisabled = this.isFocused && nextProps.disabled || nextProps.spotlightDisabled;
+			const spottableDisabled = this.isFocused && nextProps.disabled && !this.props.disabled;
```

Comment:
Does this prevent `spotlightDisabled` from working anymore?

Reply:
Nope. spottableDisabled only tracks if a previously focused comp became disabled. It'll still prevent Spottable altogether when spotlightDisabled is set.

Reply:
My concern is we also use `spottableDisabled` for setting spottability on line 346.  Why not use the same mechanism and clean up the becoming enabled part on line 202, as well?

Reply:
The idea was that at line 202, we check to see if we've become enabled after being disabled, so we can potentially restore spotlight to the component.  If `spottableDisabled` is a tracker for if a component is enabled or not, this seems like a reasonable use of that state.  That said, this does appear to work although I still can't figure out how `spotlightDisabled` works for anything other than the `isActionable` handler.

Reply:
I don't see `spottableDisabled` as tracking the overall spottability of the component which is the condition tested at L202. `spottableDisabled` would be more aptly named `spottedWhenBecameDisabled`. I'll take a cut at refactoring for readability and see if that clarifies things.

Reply:
Perhaps so.  Interestingly, if a component toggles off disabled while hovered the `onLeave` event does not fire so the component thinks it's still hovered.  If it's then restored by a component that is `spotlightDisabled` then it will gain focus even in pointer mode.  I think that's a pretty pathological set of circumstances, though.

Reply:
Hmm, I'm now not entirely sure of the mechanism for that...